src/views/promoter/auth.php

```diff
@@ -0,0 +1,52 @@
+<link href="https://fonts.googleapis.com/css?family=PT+Mono" rel="stylesheet">
+<link rel="stylesheet" href="https://use.typekit.net/pha0nnp.css">
+<link id="app" href="https://promoter.dev.tri.be/css/app.css" rel="stylesheet">
```

Comment:
🔴 This seems wrong, now that prod is up, this needs to be changed, correct? @geoffgraham @speerface